salt/beacons/__init__.py

```diff
@@ -199,13 +199,16 @@ def _update_enabled(self, name, enabled_value):
             else:
                 self.opts['beacons'][name].append({'enabled': enabled_value})
 
-    def list_beacons(self):
+    def list_beacons(self, include_pillar):
```

Comment:
IMHO, this arg needs documentation fully explaining why it is here and what it does.